src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,139 @@
+    }
+
+    //generic method to add any duke.tasks
+    public void append(Task task) {
+        this.tasks.add(task);
+    }
+
+
+    //add task
+    public void addTask(String input) throws InvalidTaskDescriptionException {
+        String[] newInput = input.split("todo ");
+        if (newInput.length < 2) {
+            throw new InvalidTaskDescriptionException("brother wake up and put a legit description can");
+        }
+        tasks.add(new Todo(newInput[1]));
+        System.out.println("Gotchu fam");
+        System.out.printf("I've added\n" + tasks.get(this.getCurrentSize()-1).toString() +
+                "\nto all the shit u need to do\n");
+        System.out.printf("shag bro now u got %d tasks\n", this.getCurrentSize());
+    }
+
+    //add event
+    public void addEvent(String input) throws InvalidTaskDescriptionException{
+        String[] nInput = input.split("/");
+        if (nInput.length < 2) {
+            throw new InvalidTaskDescriptionException("brother wake up and put a legit description can");
+        }
+        String[] newInput = input.split("/");
+        tasks.add(this.getCurrentSize(), new Event(newInput[0], newInput[1], newInput[2]));
+        System.out.println("Gotchu fam");
+        System.out.printf("I've added\n" + tasks.get(this.getCurrentSize()-1).toString() +
+                "\nto all the shit u need to do\n");
+        System.out.printf("shag bro now u got %d tasks\n", this.getCurrentSize());
+    }
+
+    //add deadline
+    public void addDeadline(String input) throws InvalidTaskDescriptionException{
+        String[] nInput = input.split("/");
+        if (nInput.length < 2) {
+            throw new InvalidTaskDescriptionException("brother wake up and put a legit description can");
+        }
+        String[] newInput = input.split("/");
+        System.out.println(Arrays.toString(newInput));
+        System.out.println(newInput[1]);
+        tasks.add(this.getCurrentSize(), new Deadline(newInput[0], newInput[1]));
+        System.out.println("Gotchu fam");
+        System.out.printf("I've added\n" + tasks.get(this.getCurrentSize()-1).toString() +
+                "\nto all the shit u need to do\n");
+        System.out.printf("shag bro now u got %d tasks\n", this.getCurrentSize());
+    }
+
+
+    //delete
+    public void deleteTask(String[] command) {
+        int taskPointer = Integer.parseInt(command[1]) - 1;
+        //duke.tasks.Task temp = duke.tasks.get(taskPointer);
+        try {
+            Task temp = tasks.get(taskPointer);
+            tasks.remove(taskPointer);
+            System.out.println("Ok bro I've removed this : ");
+            System.out.printf(temp.toString() +
+                    "\nfrom all the shit u need to do\n");
+            System.out.println("Hope you have a better life now");
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("brother look at how long ur list is first then delete lah");
+        }
+    }
```

Comment:
Consider moving the system.out.println() section to the UI.java. To increase the code quality and to let the users focus on major operations of the TaskList class.